html/elements/script.json

```diff
@@ -549,6 +549,56 @@
                 "deprecated": false
               }
             }
+          },
+          "version_parameter": {
+            "__compat": {
+              "description": "The <code>version</code> parameter of the <code>type</code> attribute.",
```

Comment:
Thanks for the bug #, it makes sense now.

No dot at the end of description.

Reply:
Done!